README.md

```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
 
 1.  Install `gnome-common`
 2.  Install the `python3-arrow`  and `python3-markdown`
```

Comment:
While we are updating the deps, this should no longer be needed (the app is now in vala)